src/wasm-binary.h

```diff
@@ -651,6 +651,7 @@ inline S32LEB binaryType(Type type) {
     case i64: ret = BinaryConsts::EncodedType::i64; break;
     case f32: ret = BinaryConsts::EncodedType::f32; break;
     case f64: ret = BinaryConsts::EncodedType::f64; break;
+    case v128: assert(false && "v128 not implemented yet");
```

Comment:
this has to be implemented for any binary read-write tests to pass, I think, does this PR not provide that?